py/tests/encoders/eye_test.py

```diff
@@ -0,0 +1,30 @@
+
+""" Unit tests for retina encoder. """
+
+import unittest
+import os
+import numpy as np
+
+from htm.encoders.eye import Eye
+
+class EyeEncoderTest(unittest.TestCase):
+  """ Unit tests for Eye encoder class. """
```

Comment:
These unit tests have no assert statements ......

Reply:
yeah, this is really just a "does not crash"/API compatible test. 

I saw really nice examples for lightning conditions (parvo) and moving objects (magno), but that would be for later.